projects/models.py

```diff
@@ -161,6 +161,13 @@ class Project(models.Model):
         null=False,
     )
 
+    democracy_project = models.BooleanField(
+        _('Democracy project'),
+        blank=False,
+        null=False,
+        default=False,
```

Comment:
In contrast to `jugendmedienschutz` we have now a `default=False`. What is the reason to handle those two differently? Does this need to be documented?

Reply:
I just added a default value because I thought that would be a good idea if the field is not nullable. I'm honestly not quite sure why `jugendmedienschutz` does not have a default value. @dhavlik I think you introduces `jugendmedienschutz` maybe you can say whether we need default values or not.